Sources/MUXSDKStatsObjc/MUXSDKStats.m

```diff
@@ -1,12 +1,6 @@
 #import "MUXSDKStats.h"
-#import "MUXSDKConstants.h"
```

Comment:
The logic in this file has been effectively moved to `MUXSDKMonitor.m`, some parts got removed completely as part of the clean-up.